components/widget/component.jsx

```diff
@@ -167,7 +167,9 @@ class Widget extends PureComponent {
         }}
         onClick={() => onClickWidget(this.props)}
       >
-        {showPlaceholder && <img src={placeholderImageURL} alt="widget" />}
+        {showPlaceholder && (
+          <img data-cy="indonesia-img" src={placeholderImageURL} alt="widget" />
```

Comment:
I understand why this was hard-coded as `indonesia`, but there are plans to make this more dynamic, right? It looks like this component has a million props so maybe we can create a dynamic `data-cy` tag using some existing prop and not require us to add another. 🤞